solutions/CSES Increasing Subsequence II.mdx

```diff
@@ -2,10 +2,24 @@
 id: cses-1748
 source: CSES
 title: Increasing Subsequence II
-author: Benjamin Qi, Óscar Garries
+author: Benjamin Qi, Óscar Garries and Timothy Gao
 ---
 
-Coordinate compress, then range sum.
+Let $dp[i]$ be the number of increasing subsequences ending on the right at $i$.
+Notice that $dp[i]$ can be calculated as the summation of all $dp[j]$ such that $j$ is in the range $[0,i]$ and $nums[j] < nums[i]$.
```

Comment:
summation of all $dp[j]$ such that $j < i$ and $x_j < x_i$
Also, please use array $x$ as defined in problem statement.